src/sql/workbench/contrib/welcome/page/browser/az_data_welcome_page.ts

```diff
@@ -26,6 +26,7 @@ export default () => `
 						<li><a href="command:notebook.command.new">${escape(localize('welcomePage.newNotebook', "New notebook"))}</a></li>
 						<li class="mac-only"><a href="command:workbench.action.files.openLocalFileFolder">${escape(localize('welcomePage.openFileMac', "Open file"))}</a></li>
 						<li class="windows-only linux-only"><a href="command:workbench.action.files.openFile">${escape(localize('welcomePage.openFileLinuxPC', "Open file"))}</a></li>
+						<li><a href="command:azdata.resource.deploy">${escape(localize('welcomePage.deploySQL', "Deploy SQL Server…"))}</a></li>
```

Comment:
I think "Install SQL Server..." terminology is more commonly used.  Has there been a discussion yet about when to use "Deploy" vs "Install" in the UI?

Reply:
There hasn't been a discussion yet but we could have one. I think Install would generally mean you run a msi. Deploy is more commonly used with docker/cluster scenarios. @alanrenmsft @dnethi  What do you think?

Reply:
@MaddyDev sounds good.  If we're scoping this feature to containers then I think this makes sense.   Though I would consider something even more descriptive such as `Deploy a SQL Server container...` or something like that.

Reply:
@kburtram , this is command is supposed to be generic enough to cover SQL Server on Docker, BDC as well as other Azure data services, it will open the dialog where you can select what to be deployed. we decided to go with SQL Server for now because we are only shipping SQL server on docker and BDC to start with. regarding to Install Vs Deploy, I think deploy makes more sense for the 2 scenarios we are covering.

this command

Reply:
Minor: why do we need `...` for this? None of the rest do. I'd suggest just checking the wording with the PMs but don't see a need to block on that

Reply:
@kevcunnane clicking on the button will not do what the link says directly, more user input is required. it is a common pattern to add ellipsis in this case.